### PR TITLE
Fix compilation using new buildchain

### PR DESCRIPTION
**Pull request by nmellado**
TODO: add implementation for option RADIUM_APP_FETCH_RADIUM

See related Radium PR : https://github.com/STORM-IRIT/Radium-Engine/pull/462

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1,15 @@
+cmake_minimum_required (VERSION 3.6)
+project(Radium-apps)
+option(RADIUM_APP_FETCH_RADIUM "Fetch, compile and link to a local copy of Radium" OFF)
+
+if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+    set (CMAKE_INSTALL_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/installed-${CMAKE_CXX_COMPILER_ID}" CACHE PATH
+            "Install path prefix, prepended onto install directories." FORCE )
+    set (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT FALSE)
+    message( "Set install prefix to ${CMAKE_INSTALL_PREFIX}" )
+endif()
```
[the remark this answers is not in the document]
Comment (MathiasPaulin) [reply CMakeLists.txt]: This is not an issue. I'll add a comment in the CMakeLists.txt file as well as for the Plugins.
To be short, Plugins are always installed in the same directory (the Radium Bundle) whereas Apps can be installed everywhere.

**Comment by MathiasPaulin** [comment]
> Regarding GUI apps, why not specifying `WIN32` along `MACOSX_BUNDLE`?

What du you mean by that ? and where ?

**Comment by hoshiryu** [comment]
`MACOSX_BUNDLE` is an option of `add_executable` that deals with MacOSX's way to manage apps. `WIN32` is another option that deals with application launch on Windows. I am wondering why are we not using this option as well?

File: CMakeLists.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (hoshiryu) [reply CMakeLists.txt]: Ok for the Plugins.
However, the check on `CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT` must be done either only once in the main `CMakeLists.txt` or separately in each app subdir, but not in both.

**Comment by nmellado** [comment]
> `MACOSX_BUNDLE` is an option of `add_executable` that deals with MacOSX's way to manage apps. `WIN32` is another option that deals with application launch on Windows. I am wondering why are we not using this option as well?

@hoshiryu could you please be more specific about the changes you would like to see ?

**Pull request merged**